wallet/wallet-rpc-lib/src/rpc/interface.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2023 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common::chain::SignedTransaction;
+
+use crate::types::{AmountString, NewAccountInfo, TransactionOptions, UtxoInfo};
+
+use super::types::{
+    AccountIndexArg, AddressInfo, AddressWithUsageInfo, BalanceInfo, BlockInfo, EmptyArgs,
+    HexEncoded, TxOptionsOverrides,
+};
+
+#[rpc::rpc(server)]
+trait WalletRpc {
+    #[method(name = "shutdown")]
+    async fn shutdown(&self) -> rpc::RpcResult<()>;
+
+    #[method(name = "best_block")]
+    async fn best_block(&self, options: EmptyArgs) -> rpc::RpcResult<BlockInfo>;
+
+    #[method(name = "create_account")]
+    async fn create_account(&self, options: EmptyArgs) -> rpc::RpcResult<NewAccountInfo>;
+
+    #[method(name = "get_issued_addresses")]
```

Comment:
We need to establish a new format for commands where we group related commands with a prefix. I'm working on that for the CLI wallet. Once the format is established, we'll use it here too.